pkg/taskrunner/errors.go

```diff
@@ -0,0 +1,8 @@
+package taskrunner
+
+import "fmt"
+
+var (
+	ErrConflict     = fmt.Errorf("conflicting commands received")
+	ErrRootFileCopy = fmt.Errorf("task not supported: copying files at the root level")
```

Comment:
No need to call formatting directives if there aren't any:

```suggestion
	ErrConflict     = errors.New("conflicting commands received")
	ErrRootFileCopy = errors.New("task not supported: copying files at the root level")
```